tools/dotcms-cli/api-data-model/src/main/java/com/dotcms/model/language/AbstractLanguage.java

```diff
@@ -21,4 +21,6 @@ public interface AbstractLanguage {
     String language();
 
     String country();
+
+    Optional<Boolean> defaultLanguage();
```

Comment:
does this really need to be an Optional ?
I mean..  I guess we're trying to figure out if a language is default or nor
therefore only one language would have that attribute set to true  everything else would come back as false
or is there for the purpose of rendering the attribute only when present?

Reply:
Done, removed unnecessary optional.